### PR TITLE
Colorize console output requiring attention

### PR DESCRIPTION
**Pull request by madarche**
It only colors the output if in a TTY

The technique used comes from https://github.com/dougwilson/nodejs-depd

### CODE REVIEW

**Comment by madarche** [comment]
@panva: OK. This will be ready in a few days. Don't close :-)

File: lib/helpers/defaults.js [hunk 1]
```diff
@@ -3,11 +3,19 @@ const epochTime = require('../helpers/epoch_time');
...
+
 const warned = new Set();
 function changeme(name, msg) {
   if (!warned.has(name)) {
     warned.add(name);
-    console.info(`NOTICE: default helper ${name} called, you should probably change it in order to ${msg}.`); // eslint-disable-line no-console
+
+    let str = `NOTICE: default helper ${name} called, you should probably change it in order to ${msg}.`;
+    if (process.stdout.isTTY) {
+      str = SET_BOLD_YELLOW_TEXT + str + RESET_ALL_ATTRIBUTES;
```
Comment (panva) [inline lib/helpers/defaults.js]: `${SET_BOLD_YELLOW_TEXT}${str}${RESET_ALL_ATTRIBUTES}` instead

**Comment by madarche** [comment]
I'll improve the code coverage.

**Comment by madarche** [comment]
@panva: I was tempted to direct all *attention* logs to `console.warn`, because that would make all those logs go down to `stderr`, and in my opinion the *changeme* logs should go to `stderr` too. Having all those logs in `stderr` is nice with process management tools like [PM2](http://pm2.keymetrics.io/docs/usage/log-management/). What do you think?

**Comment by panva** [comment]
changeme is not a directive, a must, some of them are purely informational and they need not be changed. Using dev keys on the other hand, is a different level log information.

**Pull request merged**